Bottom: df590d46e43d257735bd93f5514defd5b26cc682
Top:    8ddf10564a2d8fea149a4f29b204f8274decb177
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-12 09:22:08 -0500

Refresh of buildbot-master

---
diff --git a/ansible-playbook/staging b/ansible-playbook/staging
index 25764e80..e544fd42 100644
--- a/ansible-playbook/staging
+++ b/ansible-playbook/staging
@@ -6,7 +6,3 @@ testdaedalus.novatech-llc.com
 
 [buildsystem]
 buildsystem.novatech-llc.com
-
-[buildbot]
-testdaedalus.novatech-llc.com
-buildbot-worker.novatech-llc.com
